src/librustc_back/svh.rs

```diff
@@ -355,7 +345,7 @@ mod svh_visitor {
                     &MacInvocTT(ref path, ref _tts, ref _stx_ctxt) => {
                         let s = &path.segments;
                         assert_eq!(s.len(), 1);
-                        content(s[0].identifier)
+                        s[0].identifier.name.into()
```

Comment:
I don't like `.into()` because there's no easy way to tell what the expected type is, why aren't all of these `.as_str()`?